### PR TITLE
Ocean bbc tidy

### PR DESCRIPTION
**Pull request by aekiss**

### CODE REVIEW

**Comment by aekiss** [comment]
ocean_bc.F90 tidyup. Removed some unused lines and clarified/corrected some comments. Not expected to make any difference to code execution.

File: src/mom5/ocean_core/ocean_bbc.F90 [hunk 1]
```diff
@@ -68,6 +68,7 @@ module ocean_bbc_mod
 !  For determining bottom drag coefficient using a map of the roughness length.
 !  This approach is more relevant for coarse models
 !  than the constant roughness length used in the cdbot_law_of_wall option. 
+!  cdbot_lo <= cdbot(i,j) <= cdbot_hi.
 !  Default is cdbot_roughness_length=.false.
```
Comment (aidanheerdegen) [inline src/mom5/ocean_core/ocean_bbc.F90:72]: Should this be changed also?

Comment (aekiss) [reply src/mom5/ocean_core/ocean_bbc.F90:72]: changed to what?

Comment (aidanheerdegen) [reply src/mom5/ocean_core/ocean_bbc.F90:72]: Actually it was line 72 I thought I'd highlighted. To the same as there other changes I guess?

Comment (aekiss) [reply src/mom5/ocean_core/ocean_bbc.F90:72]: Sorry, I'm still not getting what you're suggesting. 
What needs changing in line 72: 
`!  Default is cdbot_roughness_length=.false.`
?

Comment (aekiss) [reply src/mom5/ocean_core/ocean_bbc.F90:72]: @aidanheerdegen not sure I understood what you were asking here, but are you happy for me to merge this?

Comment (aidanheerdegen) [reply src/mom5/ocean_core/ocean_bbc.F90:72]: Actually I was mistaken. Sorry for causing confusion and delay (Thomas the Tank Engine style).

Sure, merge away I say.

Comment (aekiss) [reply src/mom5/ocean_core/ocean_bbc.F90:72]: good-o - merged!

**Pull request merged**